app/src/main/java/org/gdg/frisbee/android/activity/MainActivity.java

```diff
@@ -364,11 +362,10 @@ public Chapter getSelectedChapter() {
             return mSelectedChapter;
         }
 
-        public void setSelectedChapter(Chapter chapter) {
+        public void setSelectedChapter(@NonNull Chapter chapter) {

```

Comment:
Sorry, you're checking `chapter` and not `mSelectedChapter`. I misread. Forget about my previous comment.